ogc/open-street-map-layer/README.md

```diff
@@ -14,7 +14,7 @@ When the sample opens, it will automatically display the map with the OpenStreet
 
 ## How it works
 
-1. Create an `ArcGISMap` with a `Basemap.Type.OPEN_STREET_MAP`.
+1. Create an `ArcGISMap` with a `BasemapStyle.OSM_STANDARD`.
```

Comment:
This reads a bit funky as opposed to how it was with `basemap type`. Maybe we can be a bit more verbose here?
I'm sure there will be other opinions on this, so feel free to leave it:
```suggestion
1. Create an `ArcGISMap` with a the OSM standard basemap style `BasemapStyle.OSM_STANDARD`.
```

Reply:
Thanks I've updated. Also just a general note incase it crosses anyone's mind (CC @Rachael-E ) - there is no plan to update the designs right now, that will be a future release job, so we're just to make our SDK updates as necessary for now.